Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -1361,6 +1361,32 @@ final class ProductStoreTests: XCTestCase {
         XCTAssertEqual(viewStorage.countObjects(ofType: Storage.Product.self), 0)
     }
 
+    // MARK: ProductAction.updateProducts
+
+    /// Verifies that `ProductAction.updateProducts` returns the expected `Products`.
+    ///
+    func test_updateProducts_is_correctly_updating_products() throws {
+        // Given
+        let store = ProductStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+        network.simulateResponse(requestUrlSuffix: "products/batch", filename: "products-batch-update")
+        let product = sampleProduct(addOns: [])
+        storageManager.insertSampleProduct(readOnlyProduct: product)
+
+        // When
+        var savedResult: Result<[Yosemite.Product], ProductUpdateError>?
+        waitForExpectation { expectation in
```

Comment:
nit: Same as above, we could assign `savedResult` directly instead of using an expectation here.

Reply:
Done in 18cad51.